src/main/features/directions-questionnaire/routes/hearing-location.ts

```diff
@@ -54,19 +63,27 @@ export default express.Router()
   .get(Paths.hearingLocationPage.uri, async (req: express.Request, res: express.Response, next: express.NextFunction): Promise<void> => {
     try {
       const draft: Draft<DirectionsQuestionnaireDraft> = res.locals.draft
+      //this is not right place - should be moved to tasklist builder
+      draft.document.hearingLocation = undefined
```

Comment:
should this be moved or removed?

Reply:
Yes, removed or moved or may be it's the place. When I was working on this, I didn't have complete context so I was just trying to get this page to work. Please speak to @sudheer or someone who knows DQ journey if you have questions

Reply:
@sudheer-ajax let we discuss about this.

Reply:
After discussing with @sudheer-ajax , have removed this statement to set undefined explicitly and have removed the following if else condition as well. Note - @KiranVarma02 , @robertparkinson